src/main/java/org/jahia/modules/newsletter/action/SubscribeAction.java

```diff
@@ -238,6 +260,31 @@ private boolean sendConfirmationMail(JCRSessionWrapper session, String email, JC
         return false;
     }
 
+	/**
+	* Check if templatSet has mail template
+	*/
+    private String getTemplateName(String template, JCRNodeWrapper node,  final Locale locale, String defaultTemplate){
```

Comment:
defaultTemplate is always "Jahia Newsletter", we can remove this parameter and set a static final constant in the class for the default template